docs/user-documentation/extending.md

```diff
@@ -1,13 +1,121 @@
-# Extending Islandora
+# Extending Islandora: The Islandora Cookbook
  
-Because Islandora 8 is very tightly integrated with Drupal, many use cases that required custom Islandora modules in previous versions can now be solved with a combination of configuration and modules from the Drupal community. Other use cases may still require custom code, but are not common enough, or complete enough, for that custom code to be a part of the core software. 
+Because newer Islandora is very tightly integrated with Drupal, many use cases that required custom Islandora modules in Islandora 7 and earlier can now be solved with a combination of configuration and modules from the Drupal community. Other use cases may still require custom code, but are not common enough, or complete enough, for that custom code to be a part of the core software. 
  
-Both scenarios are covered by the [Islandora Cookbook](https://github.com/Islandora-Labs/Islandora-Cookbook), which contains an annotated list of custom Islandora tools and modules developed by our community, a list of Drupal contributed modules that may be useful in an islandora context, and a set of _recipes_ which each provide a detailed outline for solving a particular use case. 
+Both scenarios are covered by the Islandora Cookbook below, which contains an annotated list of custom Islandora tools and modules developed by our community, a list of Drupal contributed modules that may be useful in an Islandora context, and a set of _recipes_ which each provide a detailed outline for solving a particular use case. 
  
 Islandora 8 is also compatible with most Drupal themes. 
+
+## Introduction
+
+In the spirit of [Islandora Awesome](https://github.com/Islandora-Labs/islandora_awesome), Islandora Cookbook is a curated list of great modules and other tools for Islandora 8. Because Islandora 8 is more tightly integrated with Drupal, most of these 'recipes' require only Drupal modules as ingredients.
+
+We offer this list for discovery, but do not officially provide support for any of these modules. 
+
+# Recipes
```

Comment:
## Recipes

(the table of contents for the page won't render if the headings aren't properly nested)